scalyr_agent/third_party/README.md

```diff
@@ -27,6 +27,12 @@ The following libraries are included:
   * [six](#six)
   * [backports.ssl-match-hostname](#ssl-match-hostname)
   * [PySocks](#PySocks)
+  * [tlslite-ng](#tlslite-ng)
```

Comment:
For consistency with other third_party libs, we should put the licenses for each of these new libraries in the `scalyr_agent/third_party/licenses` directory.